utils/watchcat/files/watchcat.sh

```diff
@@ -39,7 +39,7 @@ get_ping_family_flag() {
 	family=$1
 	case "$family" in
 	any)
-		family=""
+		family="-4"
```

Comment:
better to fix lines 221 and 226 to remove the quotes, so that ping gets to choose the family itself instead of making "any" the equivalent of IPv4-only

Reply:
So probably 140 and 145 also right ?

Reply:
> So probably 140 and 145 also right ?

Yup, thanks